crates/typst/src/export/svg.rs

```diff
@@ -26,20 +26,21 @@ const CONIC_SEGMENT: usize = 360;
 
 /// Export a frame into a SVG file.
 #[tracing::instrument(skip_all)]
-pub fn svg(frame: &Frame) -> String {
+#[comemo::memoize]
+pub fn svg(frame: &Frame) -> EcoString {
```

Comment:
Technically same here although the memory usage will be much less.

Reply:
SVGs are reasonably small, but just like PNG they could be done at a higher level which I will try and do this afternoon.